src/core/include/openvino/core/attribute_adapter.hpp

```diff
@@ -223,6 +244,44 @@ class EnumAttributeAdapterBase : public ValueAccessor<std::string> {
     AT& m_ref;
 };
 
+namespace op {
+enum class PadMode;
+enum class FillMode;
+enum class PadType;
+enum class RoundingType;
+enum class AutoBroadcastType;
+enum class BroadcastType;
+enum class EpsMode;
+enum class TopKSortType;
+enum class TopKMode;
+enum class PhiloxAlignment;
+enum class RecurrentSequenceDirection;
+}  // namespace op
+namespace element {
+enum class Type_t;
+}
+
+#ifdef _MSC_VER
+#    define OPENVINO_API_EXTERN
+#else
+#    define OPENVINO_API_EXTERN OPENVINO_API
+#endif
+
+extern template class OPENVINO_API_EXTERN EnumAttributeAdapterBase<op::PadMode>;
```

Comment:
can we place this in operation header instead? And I think it relates to all operation attributes.

Reply:
I think it should be a part of a concept of adding new operations.